src/fsharp/ParseHelpers.fs

```diff
@@ -147,42 +144,67 @@ let rec LexerIfdefEval (lookup: string -> bool) = function
 // Parsing: continuations for whitespace tokens
 //------------------------------------------------------------------------
 
+[<RequireQualifiedAccess>]
+type LexerStringStyle =
+    | Verbatim
+    | TripleQuote
+    | SingleQuote
+
+[<RequireQualifiedAccess>]
+type LexerStringKind =
```

Comment:
Can this be made a struct since it's so small?

Reply:
will do